pinot-common/src/main/java/org/apache/pinot/common/config/TableConfig.java

```diff
@@ -30,17 +30,21 @@
 import java.util.List;
 import java.util.Map;
 import javax.annotation.Nullable;
+import javax.validation.constraints.Null;
```

Comment:
Wrong import

Reply:
issue with merging, fixed. Thanks for pointing it out